main.py

```diff
@@ -184,12 +172,8 @@ async def daily(context, day):
 
     # Goes through all the players checking if they have data for that day and if they do adding to players_days
     for player in players:
-        i = 0
         #This checks they have day data
-        for item in player:
-            print(i, item)
-            i += 1
-        if (i != 5):
+        if (len(player) <= 4):
             pass
         else:
             if (day in player[4].keys()):
```

Comment:
I still don't really get this check, what is this actually checking?
If it's just checking to see if the player has a 'completion_day_level', I think all players will have that field, it might just be that they are empty, in which case it should work fine without the check

Reply:
So I had been an idiot and when you made anonymous player stuff I forgot to add the completion_day_level to their data which led to this check.  All sorted now